projects/widgets/docs/introduction.md

```diff
@@ -1,9 +1,55 @@
-<!-- focus: false -->
+# Widgets
 
-![](../assets/images/logo.png)
+A warm welcome to our Widgets documentation! 👋
 
-## Widgets
+Widgets are ready-made HTML/JS snippets that you can include in your own website to quickly provide a search page for cultural and leisure activities.
 
-A warm welcome to our Widgets documentation! 👋
+publiq provides a widget builder / wizard where you can easily configure your widget. without the need for advanced technical knowledge. During the configuration process all settings are saved on a scripttag. When embedded in a website or app, the scripttag will render the configured agenda with cultural activities.
```

Comment:
```suggestion
publiq provides a widget builder / wizard where you can easily configure your widget, without the need for advanced technical knowledge. During the configuration process all settings are saved on a scripttag. When embedded in a website or app, the scripttag will render the configured agenda with cultural activities.
```

Either this or the "without" should be capitalized since it's the start of a new sentence